### ISSUE TITLE
Latest Developer Version

### ISSUE THREAD

**Issue by fat2slow** [YePpHa/YouTubeCenter#2085]
after I downloaded the latest dev version, when ever I middle click a video it opens it in another tab but it plays for about a second and it tries to refresh the page to use the dev version and then the player just goes black and stops playing and I have to go to the tab so that the video will play

**Comment by RohithKumarSp** [comment]
yup, but what shall i do, the dev seems to have given up on the script..

**Comment by Eisys** [comment]
[Perhaps Youtube+ will suit your needs](https://github.com/ParticleCore/Particle/)

**Comment by skjnldsv** [comment]
Same here!
Please update!
